sweep/aggregator.go

```diff
@@ -534,6 +549,12 @@ func (b *BudgetAggregator) ClusterInputs(inputs InputsMap) []InputSet {
 		inputSets = append(inputSets, sets...)
 	}
 
+	// Create input sets from the exclusive inputs.
```

Comment:
> was just thinking whether we could run in the situation where we would not have enough wallet inputs to create a separate transaction for each Anchor CPFP especially because we don't use unconfirmed utxos for the sweeping system.

Yeah this has been manifested in itest - when force closes, it requires more UTXOs to create the anchor sweep. But I don't it can be improved via mempool watch tho, as even if we've seen one version of the commitment tx, it doesn't mean another version won't come in. Or more specifically - after we broadcast our local FC, we are not sure if the remote will try to also broadcast their FC. I think the only improvement we can make is, if we've seen the remote's FC first, we shouldn't attempt to sweep our anchor on our local commitment.